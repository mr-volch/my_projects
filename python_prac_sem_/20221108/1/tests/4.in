d = DivStr("LacrimosaDiesIlla")
e = DivStr("CumResurgetCreatura")
print(list(zip(list(d//2),(e//2))))
print('a', DivStr(), 'b')
print(d%2)
print(e%5)
