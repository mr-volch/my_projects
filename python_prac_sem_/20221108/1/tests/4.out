[('Lacrimos', 'CumResurg'), ('aDiesIll', 'etCreatur')]
a  b
a
tura
